### PR TITLE
kit publish

### PR DESCRIPTION
**Pull request by nick1udwig**

### CODE REVIEW

File: src/kit/publish.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+short: `kit p`
+
+`kit publish` creates entries in the Kimap, publishing the given package according to the `app_store`s protocol.
+It can also be used to update or unpublish previously-published packages.
+`kit publish` directly writes to the Kimap: it does not interact with a Kinode.
+
+## Example Usage
+
+```bash
+# Publish a package on the real (live) network.
```
Comment (dr-frmr) [inline src/kit/publish.md:12]: i would add a comment here that currently this is optimism mainnet

File: src/kit/publish.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  -k, --keystore-path <PATH>
+          Path to private key keystore (choose 1 of `k`, `l`, `t`)
+  -l, --ledger
+          Use Ledger private key (choose 1 of `k`, `l`, `t`)
+  -t, --trezor
+          Use Trezor private key (choose 1 of `k`, `l`, `t`)
+  -u, --metadata-uri <URI>
+          URI where metadata lives
+  -r, --rpc <RPC_URI>
+          The ETH RPC WebSockets URI
```
Comment (dr-frmr) [inline src/kit/publish.md:42]: comment here that RPC should be for optimism mainnet

File: src/kit/publish.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Publish the metadata for the package in this directory.
+
+### `--metadata-uri`
+
+short: `-u`
+
+The URI hosting the `metadata.json`.
+You must place the `metadata.json` somewhere public before publishing your package on Kimap.
+A common place to host `metadata.json` is on your package's Github repo.
```
Comment (dr-frmr) [inline src/kit/publish.md:67]: i think it's good to have a reminder here that the link should be static, i.e. the github asset permalink and not a branch-specific URL where underlying content can easily change

**Comment by nick1udwig** [comment]
Thanks for the review -- addressed your comments in 7538462

**Pull request merged**